app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -210,16 +215,31 @@ fun clearHikesFromMap(mapView: MapView, userLocationMarker: Marker?) {
   mapView.invalidate()
 }
 
+/**
+ * Properties to initialize the map with. These properties are used to set the initial state of the
+ * map, such as the zoom level, the center, and the limits of the zoom.
+ *
+ * @param mapInitialZoomLevel The initial zoom level of the map
+ * @param mapMaxZoomLevel The maximum zoom level of the map
+ * @param mapMinZoomLevel The minimum zoom level of the map
+ * @param mapInitialCenter The initial center of the map
+ */
+data class MapInitialValues(
```

Comment:
Comment: I like this improvement, but is there a reason why you changed it? I mean a reason related to the PR?

Reply:
It's not really related but I had Sonar complains about to many arguments for MapScreen function

Reply:
Good improvemet